packages/connector-jsdom/src/evaluate-runner.ts

```diff
@@ -28,7 +40,19 @@ const run = async (data: { source: string }) => {
         debug(err);
     });
 
-    const jsdom = await JSDOM.fromURL(url, {
+    let html = '';
+
+    try {
+        const networkData = await requester.get(url);
```

Comment:
Probably worth a comment here or before the `run` function explaining we need to request the HTML separately instead of just using `JSDOM.fromURL` in order to respect user options to ignore HTTPS errors.